Plots/Contours/NCL_polar_1_lg.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
```

Comment:
As far as I understand, `Plots/Contours/NCL_polar_1_lg.py` is already being implemented within another PR #75  by the branch **NCL_polar_1_lg.py**. 

Shoul we keep this file in this PR, or otherwise, should you remove it from this PR?

Reply:
I just saw I have the wrong file uploaded here. Let me check into resolving by uploading the correct file here instead

Reply:
Actually I see what the problem is, how would I go about removing the other file? This also happens in the proj_3 PR as well but it includes polar_1 and proj_2 so I will have to do the same thing there

Reply:
It looks like you were able to delete the wrong file. Great!